tests/suites/manual/deploy_manual_aws.sh

```diff
@@ -1,3 +1,42 @@
+create_vpc() {
+	vpc_id=$(aws ec2 create-vpc --cidr-block 10.0.0.0/28 --query 'Vpc.VpcId' --tag-specifications 'ResourceType=vpc,Tags=[{Key=Name,Value=manual-deploy}]' --output text)
```

Comment:
can we name this vpc something unique to this job, and also have a tag we can query to cleanup these in the z-aws-cleanup job

Reply:
by unique, I mean random, if we have two of these running at the same time I don't want errors because its being tested by two people/jobs at the same time.